js/components/pages/PassagePage.react.js

```diff
@@ -20,6 +21,9 @@ class PassagePage extends Component {
       previousButton = <button className="previous" onClick={() => { dispatch(asyncPreviousVerse()) }}>Previous</button>
     }
 
+    const audioUrl = "http://www.esvapi.org/v2/rest/passageQuery?key=IP&passage=" +
```

Comment:
I'll leave this as IP-based access then until we have a plan for using the API key.